zwave_js_server/util/node.py

```diff
@@ -86,7 +86,11 @@ async def async_set_config_parameter(
     # Validate that new value for range configuration parameter is within bounds
     max_ = zwave_value.metadata.max
     min_ = zwave_value.metadata.min
-    if zwave_value.configuration_value_type == ConfigurationValueType.RANGE and (
+    value_ = (
```

Comment:
We set `min_` and `max_` for convenience because they are referenced multiple times below. `value_` is only referenced once, and it's name implies that it represents a value but it doesn't. Can we collapse this into the if statement and remove `value_` altogether?

Reply:
Doing so gave me a linting error, too many bools (6/5).
I'm up for renaming the variable to something else.

Reply:
I've renamed it to `check_` since it's technically checking the value types.